Product/pom.xml

```diff
@@ -292,6 +292,7 @@
                 <groupId>xerces</groupId>
                 <artifactId>xercesImpl</artifactId>
                 <version>2.10.0</version>
+                <scope>provided</scope>
```

Comment:
Scope of "provided" is not the right way to do this, we should exclude the jars where they are transitive dependencies.